plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -231,7 +232,7 @@ public static GlueHiveMetastore createTestingGlueHiveMetastore(java.nio.file.Pat
                 glueConfig,
                 directExecutor(),
                 new DefaultGlueColumnStatisticsProviderFactory(directExecutor(), directExecutor()),
-                createAsyncGlueClient(glueConfig, DefaultAWSCredentialsProviderChain.getInstance(), Optional.empty(), stats.newRequestMetricsCollector()),
+                createAsyncGlueClient(glueConfig, DefaultAWSCredentialsProviderChain.getInstance(), Optional.empty(), OpenTelemetry.noop(), stats.newRequestMetricsCollector()),
```

Comment:
or maybe we should plug in real telemetry here?
unless it's redundant because there already is something?

cc @nineinchnick @electrum @wendigo

Reply:
I have an airlift change that allows to plug-in custom span processor: https://github.com/airlift/airlift/pull/1082

This would make this PR smaller I think

Reply:
we should.

I have an airlift change that allows to plug-in custom span processor: https://github.com/airlift/airlift/pull/1082

This would make this PR smaller I think

Reply:
do if i am using Glue metastore, currently the requests aren't traced?
or is there something in AWS SDK that we configure (or could configure) to address all the AWS clients?

Reply:
Glue requests weren't traced. I confirmed it before making this PR.

Reply:
There is io.opentelemetry.instrumentation:opentelemetry-aws-sdk-2.2 so I guess it should be possible: https://aws-otel.github.io/docs/getting-started/java-sdk/manual-instr

Reply:
I changed the optional RequestHandler2 to Set for avoiding a new OpenTelemetry argument.